modules/sdk-core/src/bitgo/wallet/wallet.ts

```diff
@@ -1614,10 +1614,11 @@ export class Wallet implements IWallet {
       if (typeof this.baseCoin.signWithCustomSigningFunction === 'function') {
         return this.baseCoin.signWithCustomSigningFunction(params.customSigningFunction, signTransactionParams);
       }
-      const userKey = await this.baseCoin.keychains().getKeysForSigning({ wallet: this })[0];
+      const keys = await this.baseCoin.keychains().getKeysForSigning({ wallet: this });
+      const derivationSeed = keys[0] ? keys[0].derivedFromParentWithSeed : undefined;
```

Comment:
maybe find by `source === user` instead of index?

Otherwise just do `derivationSeed: userKey?.derivedFromParentWithSeed` in line 1620

Reply:
Did not find source in the type of key in the sdk, im assuming WP provides it but we should do the typing of this as a follow up.